lib/Sema/CSSimplify.cpp

```diff
@@ -3048,7 +3048,7 @@ bool ConstraintSystem::repairFailures(
     // explicit call.
     if (fnType->getNumParams() > 0) {
       auto anchor = simplifyLocatorToAnchor(getConstraintLocator(locator));
-      if (!anchor.is<Expr *>())
+      if (!anchor || !anchor.is<Expr *>())
```

Comment:
Ah, you can use `!isExpr(anchor)` here instead, that would do a null check internally. Looks like I missed this place while doing switch to `ASTNode` :(

Reply:
Ah yeah nice, in fact let's use `getAsExpr` exclusively here